from exceptions.exceptions import DataPathException